clientd3d/intrface.c

```diff
@@ -27,6 +27,8 @@ static Bool interface_created = False;  /* True iff interface has been created *
 extern font_3d				gFont;
 extern d3d_driver_profile	gD3DDriverProfile;
 
+static DWORD last_text_command;
```

Comment:
This name makes it sound like it's the last text command that was entered.  last_text_command_time, and add a comment?